### DISCUSSION TITLE
sort streams into public lists by FPS?

### DISCUSSION THREAD

**Discussion by Owyn** [iptv-org/community#106] [Ideas]
Best stream quality is currently selected between resolutions ( 720p and 1080p for e.g.),
- would be nice to also select by the best FPS quality because 25fps streams and ~30fps ones are usually clearly distinguishable in quality, and there are even  channels broadcasting in 50fps+

**Comment by BellezaEmporium** [comment]
> Best stream quality is currently selected between resolutions ( 720p and 1080p for e.g.),
> 
>     * would be nice to also select by the best FPS quality because 25fps streams and ~30fps ones are usually clearly distinguishable in quality, and there are even  channels broadcasting in 50fps+

Would be useless for two reasons.

1) We mainly attach ourselves to resolution for quality of image.
2) The way the number of images per second are dealt don't mean much on stream.

**Reply by Owyn** [reply]
> The way the number of images per second are dealt don't mean much on stream.

except it does

Have you ever watched a 60fps video?

And most streams are 25 fps barely-ok streams while some are good quality ones with 29.97 fps which is clearly seen

**Reply by BellezaEmporium** [reply]
Does the end user care about the FPS of a video ? As long as it"s watchable, it's fine.

**Reply by Owyn** [reply]
>  As long as it"s watchable, it's fine.

Then why do you even sort by quality? just leave 240p in the list instead of 1080p if it's fine

I just want to improve the already existing feature 

Because as a viewer I care about the fps (and overall quality of the stream)

**Comment by freearhey** [comment]
Can you give an example of a channel that broadcasts in the same resolution but with different frame rates?

**Reply by Owyn** [reply]
it's not channels themselves but "re-broadcasters" who reduce the stream quality I believe

example:
1080p@25fps: `https://strm.yandex.ru/kal/rbc/rbc0.m3u8`  (geo-blocked btw)
1080p@50fps: `https://live-rbc.gcdn.co/gcore/rbctvhd.m3u8`

**Comment by freearhey** [comment]
Done in #7478
